jib-maven-plugin/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file.
 ## [unreleased]
 
 ### Added
+- Can now decrypt proxy configuration in `settings.xml`. ([#1369](https://github.com/GoogleContainerTools/jib/issues/1369))
```

Comment:
I think this should be `_a_ proxy configuration` or `proxy configuration_s from_`?